lib/utilities/reportHandler.js

```diff
@@ -34,13 +34,15 @@ function hasWSOptionalDependencies() {
 }
 
 function getCapabilities() {
+    const localVolumeCap = process.env.LOCAL_VOLUME_CAPABILITY;
```

Comment:
Not a fan of using JSON.parse to get a bool, I would just do a string check with two changes.
```suggestion
    const localVolumeCap = process.env.LOCAL_VOLUME_CAPABILITY || 'true';
```
Default to a true string

Reply:
can this just be set to a `bool` without the parsing?
also would prefer a specific value vs all truthy

```suggestion
    const localVolumeCap = process.env.LOCAL_VOLUME_CAPABILITY === '1'; // or 'true'
```"

Reply:
yea but I prefer to be a little more lenient to account for capitals such that 'True' and 'true' are both ok, some people like TRUE or true for values. I know python has a isTruthyVar or something that supports most common true and false values like true, True, 1, yes, y, etc. Could also work. Found it: https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool

Reply:
a truthy set can work too. the specific case I wanted to avoid was '0', which would resolve to true.
~I believe in this project, we are using `0` or `1`~ (it's a mix)

Reply:
0 would still be false when you take into account my other suggestion of `.toLowerCase() === 'true'`